samples/java/src/main/java/com/contoso/kafka/FunctionOutput.java

```diff
@@ -0,0 +1,28 @@
+package com.contoso.kafka;
+
+import java.util.*;
+import com.microsoft.azure.functions.annotation.*;
+import com.microsoft.azure.functions.*;
+
+import com.microsoft.azure.functions.kafka.annotation.*;
+import com.microsoft.azure.functions.kafka.*;
+
+
+import java.util.Optional;
+
+public class FunctionOutput {
+    @FunctionName("KafkaInput-Java")
+    public HttpResponseMessage input(
+            @HttpTrigger(name = "req", methods = {HttpMethod.GET, HttpMethod.POST}, authLevel = AuthorizationLevel.ANONYMOUS) HttpRequestMessage<Optional<String>> request,
+            @KafkaOutput(topic = "my-confluent-topic", brokerList="localhost:31090")  OutputBinding<String> output,
```

Comment:
This is not big deal, However, you have a recommendation for the development environment. topic name and broker list is my environment. It might be better to fit the C# sample or python sample, however, if you don't change it it also ok.

Reply:
good points. i will update it to be closer to the C# sample